CMakeLists.txt

```diff
@@ -177,6 +177,21 @@ if(WITH_ROS_SUPPORT)
   )
 endif()
 
+if(WITH_ROS_SUPPORT)
+  AddProject(gram_savitzky_golay
+    GITHUB arntanguy/gram_savitzky_golay
+    GIT_TAG origin/master
+  )
+  set(MC_STATE_OBSERVATION_ROS_OPTION "-DWITH_ROS_OBSERVERS=ON")
+else()
+  set(MC_STATE_OBSERVATION_ROS_OPTION "-DWITH_ROS_OBSERVERS=OFF")
+endif()
+AddProject(mc_state_observation
+  GITHUB jrl-umi3218/mc_state_observation
+  CMAKE_ARGS ${MC_STATE_OBSERVATION_ROS_OPTION}
+  DEPENDS mc_rtc
```

Comment:
We need to have the dependency on gram_savitzky_golay as well when ROS_SUPPORT is on

Reply:
Done in https://github.com/mc-rtc/mc-rtc-superbuild/pull/5/commits/32223ab97334c221cc4f9bc83db2acba6c4f9656